node/sc/bridgepeer.go

```diff
@@ -48,15 +48,16 @@ const (
 // BridgePeerInfo represents a short summary of the Klaytn Bridge sub-protocol metadata known
 // about a connected peer.
 type BridgePeerInfo struct {
-	Version int    `json:"version"` // Klaytn Bridge protocol version negotiated
-	Head    string `json:"head"`    // SHA3 hash of the peer's best owned block
+	ProtocolVersion string `json:"protocolversion"` // Klaytn Bridge protocol version negotiated
```

Comment:
I am not sure it is okay to change the "version" to "protocolversion". Did you check this is working well with previous version of klaytn??

Reply:
Why do you think this renaming causes a problem? For example, version v1.7.3 uses `Version` variable as version representation, while the node v1.8.0 uses `ProtocolVersion` variable. Could you give more specific context if I misunderstood your concerns?

Reply:
What if a klayn-v1.7.3 node is trying to connect a klaytn-v1.8.0 node? Is this okay? For Baobab and Cypress node, it should not be the problem.

Reply:
Suppose that `kscn` node uses v1.7.3 in the SC network and `ken` with this PR version in the baobab or cypress network. Each node in its own network cannot establish a P2P connection since their node versions are different from each other. This PR was designed to guarantee to have the same internal bridge logic to avoid surprise malfunctioning due to the different implementation. 

If the compatibility between old and new bridge implementations is necessary, I think two options can be considered.
1. The node must be updated with the latest version. (Current PR's one)
2. The node must be updated with a minimum compatible node version.

If your concern is neither first nor second, both protocol version and node version check should be reverted again.

Reply:
@hyunsooda Do we need to change the name of this?

Reply:
Restored.